examples/layout/layout.ts

```diff
@@ -15,7 +15,7 @@ import footerTemplate from './footer.xml';
 import { ShadowNode } from '../../packages/plugin-shadow/src/ShadowNode';
 import { MetadataNode } from '../../packages/plugin-metadata/src/MetadataNode';
 import { parseEditable, createEditable } from '../../packages/utils/src/configuration';
-import { Fullscreen } from '../../packages/plugin-fullsreen/src/Fullscreen';
+import { Fullscreen } from '../../packages/plugin-fullscreen/src/Fullscreen';
```

Comment:
Bye bye "fullsreen", we will miss you. Your legacy will live on. 🚀

Reply:
:trollface: :-D